browser/scss/home/main.scss

```diff
@@ -13,18 +13,70 @@
 .home-container {
 	margin: 0 auto;
 	width: 80%;
-	margin-top: 30px;
+  margin-top: 70px;
+}
+
+.graph-home {
+  margin: 50px 0;
+  background-image: url('graph-background.png');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  height: 500px;
```

Comment:
Are we going to have hard-coded pixel values in the final product or will everything be relative to screen size / resizable?